grapple/types/redirects.py

```diff
@@ -1,31 +1,52 @@
+import json
+
+from typing import Optional
+
 import graphene
 
-from django.conf import settings
 from wagtail.contrib.redirects.models import Redirect
+from wagtail.models import Page, Site
 
 from .pages import get_page_interface
 
 
 class RedirectType(graphene.ObjectType):
```

Comment:
Can you please rename this to `RedirectObjectType` to be consistent with other types? And perhaps make this nicer to use by specifying a name too:

```suggestion
class RedirectObjectType(graphene.ObjectType):
    ... fields ...
    
    class Meta:
        name = 'Redirect'
```

`RedirectType` isn't as clear.

Reply:
Addressed in 6a7294b264f3584e04fdc0d279e96a9e8c048a37